### ISSUE TITLE
add option to create a default image when lazyload is active 

### ISSUE THREAD

**Issue by tillschweneker** [sitegeist/sms-responsive-images#42]
The idea is to show the user a preview of the image until the real image is loaded.
The image tag could be rendered like this:
`<img class="image-embed-item lazyload" src="/fileadmin/_processed_/3/7/image-1_really_small.jpg" data-srcset="/fileadmin/_processed_/3/7/image-1_1d4a98938c.jpg 800w, /fileadmin/_processed_/3/7/image-1_45daa42d81.jpg 1200w, /fileadmin/_processed_/3/7/image-1_c9ff912115.jpg 300w" sizes="(min-width: 800px) 800px, 100vw" width="800" height="500" alt=""> ` where the image linked in `src` is just 8x5 px.

This way the html would be valid, the user would see a blurry preview and it could be easily animated to the real image with a lazyload function.

The size of the lazyload fallback image could be set via the options, e.g.
`# set the image size of the fallback image when lazyload is active in percent`
`lib.contentElement {
  settings.tx_smsresponsiveimages {
    lazyloadFallbackImageSize = 1 # for 1%
  }
}`

**Comment by moe2k** [comment]
Hi @tillschweneker did you find a solution for this? I'm having the same problem.

**Comment by s2b** [comment]
I just implemented this feature in Branch feature/lazyloadPlaceholder. Could you test if this works for your use case?

Configuration for content images in Constants:

```
tx_smsresponsiveimages {
  # Placeholder images are 20 pixels wide
  placeholder.size = 20
  # Set to 1 if the placeholder should be rendered as base64 data uri
  placeholder.inline = 0
}
```

There are also two new arguments for both ViewHelpers:

`<f:image ... placeholderSize="20" placeholderInline="1" />`

ToDo on my part:

* Documentation
* Tests

**Comment by moe2k** [comment]
Cool! Tested the sms:image viewhelper both with and without placeholderSize and placeholderInline, as well as the typoscript setting for overriding fluid_styled_content. Everything worked as expected!

**Comment by s2b** [comment]
Today I released v1.3.0 of the extension which includes the feature.

**Comment by moe2k** [comment]
Thank you very much, thumbs Up!!
